src/state/queries/post-feed.ts

```diff
@@ -410,10 +409,10 @@ function createApi({
         userInterests,
       })
     } else {
-      if (useBaseFollowingFeed) {
-        return new FollowingFeedAPI({getAgent})
-      } else {
+      if (enableFollowingToDiscoverFallback) {
```

Comment:
I found flipping this a little easier to think about, like an extra feature whose natural state is "off". Then `=== 0` condition above seems pretty plain, and doesn't need to be inverted.